### PR TITLE
add post on upcoming fosdem 2023

### PR DESCRIPTION
**Pull request by fkr**

### CODE REVIEW

File: _i18n/en/_posts/blog/2023-02-01-fosdem-2023-sovereign-cloud-devroom.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+There are numerous other devrooms that touch topics that are relevant to the SCS community - to be honest, basically all of them do, so this is just an excerpt. ;)
+
+* [Containers](https://fosdem.org/2023/schedule/track/containers/)
+* [Virtualization and IaaS](https://fosdem.org/2023/schedule/track/virtualization_and_iaas/)
+* [Software Defined Storage](https://fosdem.org/2023/schedule/track/software_defined_storage/)
+* [CI/CD](https://fosdem.org/2023/schedule/track/continuous_integration_and_continuous_deployment/)
+* [Software Bill of Materials](https://fosdem.org/2023/schedule/track/software_bill_of_materials/)
+
+If you now have the urge to develope [FOMO](https://en.wikipedia.org/wiki/Fear_of_missing_out), no worries: All the talks are recorded and available for later consumption!
```
Comment (itrich) [inline _i18n/en/_posts/blog/2023-02-01-fosdem-2023-sovereign-cloud-devroom.md:40]:
```suggestion
If you now have the urge to develop [FOMO](https://en.wikipedia.org/wiki/Fear_of_missing_out), no worries: All the talks will be recorded and available for later consumption!
```

**Review by itrich — CHANGES_REQUESTED** [review]
Just some minor typos. I also added a preview banner. Feel free to drop it if you don't like it.

**Comment by fkr** [comment]
thanks @itrich - lovely!

**Comment by fkr** [comment]
(and sorry for missing the symlink)

**Review by itrich — APPROVED** [review]
LGTM

**Pull request merged**